Exercises/chapter-10.txt

```diff
@@ -0,0 +1,7 @@
+1. Why is it important to protect both the edit and update actions?
+Because these are actions that can change data in your database.
+
+
+
+2. Which action could you more easily test in a browser?
+The edit, since it can be accessed using a URL.
```

Comment:
yes, b/c when you type a URL and hit enter, the browser does a GET request.  Whereas update needs a POST request, which is tougher to get your browser to do.

FYI: Postman is nice tool for testing all the HTTP actions